datastore.py

```diff
@@ -326,16 +342,34 @@ def GetOrInsertDefault():
 
   @staticmethod
   def InsertDefault():
+    """Insert the DV entity with default content into the datastore.
+    """
```

Comment:
If we just have the one-liner summary, the closing """" can be on the same line.
"""Insert the DV entity with default content into the datastore."""